__tests__/components/editor/InputLookupQA.test.js

```diff
@@ -82,4 +114,38 @@ describe('<InputLookup />', () => {
     const propertyRemark = wrapper.find('label > PropertyRemark')
     expect(propertyRemark).toBeTruthy()
   })
+
+  //Institute wrapper with multiple lookup options
+  const multipleWrapper = shallow(<InputLookup.WrappedComponent {...p2Props} handleSelectedChange={mockFormDataFn} />)
+  it('should pass multiple lookup results in state with search event', () => {
+   const event = (wrap) => {
+	   wrap.setState({options: multipleResults})
+   }
+   multipleWrapper.find('#lookupComponent').simulate('search', event(multipleWrapper))
+	   expect(multipleWrapper.state().options[0]).toEqual(multipleResults[0]) 
+	   expect(multipleWrapper.state().options[1]).toEqual(multipleResults[1]) 
+
+  })
+  //Headers expected
+
+  it('should show menu headers with lookup source labels and values in the dropdown when provided results', () => {
+	  console.log("Wrapper instance");
+	  const instance = multipleWrapper.instance();
+	  const menuWrapper = shallow(instance.renderMenuFunc(multipleResults, p2Props));
+	  //console.log(menuWrapper.childAt(0).render().html());
```

Comment:
I would remove these commented-out log statements.

Reply:
I've removed the commented out statements as well the first console.log which was just there for debugging the test.